### ISSUE TITLE
Reaction Role Section

### ISSUE THREAD

**Issue by DataPirate007** [AAADevs/alpha-bot#24]
### About
This is for allowing the bot to add roles by the reactions of the users

### Reference Link
https://github.com/GhostDark123/Zira

### Commands Needed
Remove the Trello integration command
![image](https://user-images.githubusercontent.com/66989464/89402196-1e9b8000-d734-11ea-9b2e-00de18cccbb2.png)

### Examples
![image](https://user-images.githubusercontent.com/66989464/89402348-51de0f00-d734-11ea-89d4-f573136f553a.png)
![image](https://user-images.githubusercontent.com/66989464/89402380-60c4c180-d734-11ea-82d4-3edea7f1238c.png)

**Comment by SudhamJayanthi** [comment]
This is Python Discorder

**Comment by SudhamJayanthi** [comment]
@Te

**Comment by Pyroseza** [comment]
@SudhamJayanthi I don't know of any, whoever linked that deleted their account so we can't ask them, however I did find the official documentation of the Zira bot: https://zira.discorddocs.com/docs/en/start/

**Comment by Pyroseza** [comment]
So I looked at the link in the desc, something I noticed is, yes it is a javascript bot and the original repo it was forked from is archived is a correct analysis however the point is not to just copy code and paste it in our bot, but to look at the functionality that we like and make our own version of it in Python.

**Comment by solidassassin** [comment]
This is quite simple to do. Just a `on_reaction_add` or `on_raw_reaction_add` even with some if statements and action taking after.

**Comment by SudhamJayanthi** [comment]
> @SudhamJayanthi I don't know of any, whoever linked that deleted their account so we can't ask them, however I did find the official documentation of the Zira bot: https://zira.discorddocs.com/docs/en/start/

I’ve been away for a while, so haven’t seen your replies. Thank you for the documentation.

**Comment by SudhamJayanthi** [comment]
> So I looked at the link in the desc, something I noticed is, yes it is a javascript bot and the original repo it was forked from is archived is a correct analysis however the point is not to just copy code and paste it in our bot, but to look at the functionality that we like and make our own version of it in Python.
> 
> 
> 
> 

Yeah definitely, copying is not my intention too but I found it harder to understand because I’m new to both discord bots & JavaScript. I’ll try to find a better resource & also will lookup the documentation you shared 👍🏻

**Comment by SudhamJayanthi** [comment]
> This is quite simple to do. Just a `on_reaction_add` or `on_raw_reaction_add` even with some if statements and action taking after.

Ohk ! Thanks for the suggestion. I didn’t know that, will check it !
